templates/build.gradle

```
@@ -138,6 +138,7 @@ dependencies {
     }
 
     testImplementation "org.junit.jupiter:junit-jupiter-api:5.6.2"
+    testImplementation "org.junit.jupiter:junit-jupiter-params:5.5.2"
```

oh this is using 5.6 for the other junit-jupiter stuff, so this, but yes

```suggestion
    testImplementation "org.junit.jupiter:junit-jupiter-params:5.6.2"
```